src/components/Dropdown/Dropdown.tsx

```diff
@@ -0,0 +1,214 @@
+import * as React from 'react';
+import { themr, ThemedComponentClass } from 'react-css-themr';
+import { autobind } from '@shopify/javascript-utilities/decorators';
+import { createUniqueIDFactory } from '@shopify/javascript-utilities/other';
+import { findFirstFocusableNode } from '@shopify/javascript-utilities/focus';
+import { addEventListener, removeEventListener } from '@shopify/javascript-utilities/events';
+
+import DropdownItem from './DropdownItem';
+import { classNames } from '@shopify/react-utilities/styles';
+import { DROPDOWN } from '../ThemeIdentifiers';
+import * as baseTheme from './Dropdown.scss';
+import { Keys } from '../../types';
+import Button from '../Button';
+
+export type Direction = 'up' | 'down' | 'left' | 'right';
+
+export interface DropdownItemProps {
+  children?: React.ReactNode;
+  disabled?: boolean;
+  divider?: boolean;
+  header?: boolean;
+  onClick?(): void;
+}
+
+export interface Props {
+  disabled?: boolean;
+  children?: string;
```

Comment:
@shyamk9 No need to add `children` prop in interface, its a default prop which comes with every component, so we don't have to give it a type